src/components/LoanCards/KivaClassicBasicLoanCardExp.vue

```diff
@@ -229,6 +230,10 @@ export default {
 		categoryPageName: {
 			type: String,
 			default: '',
+		},
+		showMoreLink: {
```

Comment:
This was previously used to toggle the "View loan" version of the button that goes to BP. It would be nice to have that present so this loan card could be used in contentful landing pages if it becomes our new default...

Reply:
We'll be adding the "view loan" CTA next sprint, since we're experimenting with the new loan card on the home page. Looks like we'll have to port this card to CPS...

Reply:
Removed, thanks both!